src/main/java/Game.java

```diff
@@ -24,23 +18,11 @@
     private final ActionManager actionManager;
 
     Game() {
-        GameObjectManager gameObjectManager = new GameObjectManager();
-
-        this.physicManager = new PhysicManager(new Rectangle(new Vector(0, 0), new Vector(100, 100)),
-                gameObjectManager.getPhysicObjectManager());
-        this.actionManager = new ActionManager();
-
-        InteractionRuleBase interactionRuleBase = new InteractionRuleBase();
-
-        interactionRuleBase.addRule(new InteractionType(Type.TANK, Type.BULLET),
-                new TankBulletInteractionRule(gameObjectManager));
-        interactionRuleBase.addRule(new InteractionType(Type.TANK, Type.BONUS),
-                new BonusTankInteractionRule(gameObjectManager));
-
-        this.interactionProcessor = new InteractionProcessor(interactionRuleBase);
-
-        GameInitializer gameInitializer = new GameInitializer();
-        gameInitializer.initialize(gameObjectManager, physicManager);
+        GameFacade gameFacade = new GameFacade();
+        gameFacade.createGame(new Rectangle(new Vector(0, 0), new Vector(100, 100)));
```

Comment:
when you name method as "createGame()" then i expect it to return Game object. Please name it better